tests/jenkins.test.ps1

```diff
@@ -0,0 +1,163 @@
+Describe "Jenkins Configuration" {
+    Context "Installation Integrity" {
+        BeforeAll {
+            $jenkins = choco list -lo -r | ConvertFrom-Csv -Delimiter '|' -Header Package,Version | Where-Object Package -eq 'jenkins'
+            $service = Get-Service jenkins
```

Comment:
Personally in my tests I like using initial capitals on anything that is defined in `BeforeAll` and used in tests, then inside of `It` blocks I use lower case to differentiate from "used elsewhere" and "just used here".

And on that note, regardless if you take my suggestion, I would say keep it consistent. Here they're lower case, but line 9 has `$Jenkins`.